autopub/prepare_release.py

```diff
@@ -85,6 +87,7 @@ def prepare_release():
         f.write(f"{VERSION_HEADER * len(new_version_header)}\n\n")
 
         f.write(release_changelog)
+        append_github_contributor(f)
```

Comment:
Can we add a check here? or make more generic... 
If I don't use GitHub in the stack this will make a request anyway... 

Think 
1. this can be more generic: if I don't use GitHub we can pick up the user form the commit 
2. Make it optional: in an organization maybe is not so important to know who did the change...